packages/core/src/external/cda/remove-b64.ts

```diff
@@ -37,18 +47,47 @@ export function removeBase64PdfEntries(payloadRaw: string): {
         if (comp.section.entry) {
           //eslint-disable-next-line @typescript-eslint/no-explicit-any
           comp.section.entry = toArray(comp.section.entry).filter((entry: any) => {
-            const mediaType = entry.act?.text?.["_mediaType"]?.trim().toLowerCase();
-            if (
-              (BINARY_MIME_TYPES.includes(mediaType) || mediaType == undefined) &&
-              (entry.act?.text?.["_representation"]?.trim().toLowerCase() ===
-                b64Representation.toLowerCase() ||
-                entry.organizer?.component?.observationMedia?.value?.["_representation"]
-                  ?.trim()
-                  .toLowerCase() === b64Representation.toLowerCase())
-            ) {
-              isRemovedEntries = true;
-              b64Attachments.push(entry);
-              return false;
+            if (isConcernActEntry(entry)) {
+              const act = entry.act;
+              const mediaType = act.text?.["_mediaType"]?.trim().toLowerCase();
+              if (
+                (mediaType == undefined || BINARY_MIME_TYPES.includes(mediaType)) &&
```

Comment:
You should probably also do a `isBinaryMimeType` that handles the trim Lowercase step like you did`isB64....` -- it's a good habit to form of wrapping the raw types in these methods that create the access to the underlying const.